sdks/python/keelson/__init__.py

```diff
@@ -121,5 +123,5 @@ def get_tag_encoding(tag: str) -> str:
     return _TAGS[tag]["encoding"]
 
 
-def get_tag_description(tag: str) -> str:
-    return _TAGS[tag]["description"]
+def get_tag_schema(tag: str) -> str:
+    return _TAGS[tag]["schema"]
```

Comment:
Is this safe? What if the the tag doesn't exist? Maybe either type tag to other than `str` or change return type to `Optional[str]`?